assets/js/admin-settings/rule.jsx

```diff
@@ -0,0 +1,145 @@
+/* global wp, wpLocalizedData */
+import {
+  Button,
+  SelectControl,
+  TextControl,
+  ToggleControl,
+} from '@wordpress/components';
+import { __ } from '@wordpress/i18n';
+import React, { useEffect, useState } from 'react';
+import { useSelect } from '@wordpress/data';
+import { ruleCard } from './styles';
+
+
+const Rule = ({
+  field,
+  loading,
+  newRule,
+  onAdd,
```

Comment:
Recommend handling add via the main form. Put a button at the bottom to add a new row. Let this component be for editing and deleting only. When a new item is added, this component will take over and allow the user to set values, and it will initialize to the defaults (empty string for everything except term ID, which is 0).